packages/logger/src/logger/ogma.ts

```diff
@@ -154,15 +164,21 @@ export class Ogma {
     if (typeof message === 'object' && !(message instanceof Error)) {
       message = '\n' + JSON.stringify(message, this.circularReplacer(), 2);
     }
-    application = this.toStreamColor(application || this.options.application, Color.YELLOW);
+    const { logHostname, logApplication, logPid } = this.options;
+
     context = this.toStreamColor(context || this.options.context, Color.CYAN);
     correlationId &&= this.wrapInBrackets(correlationId);
 
-    const hostname = this.toStreamColor(this.hostname, Color.MAGENTA);
     const timestamp = this.wrapInBrackets(this.getTimestamp());
-    return `${timestamp} ${formattedLevel} ${hostname} ${application} ${this.wrapInBrackets(
-      this.pid.toString(),
-    )} ${correlationId} ${context} ${message}`;
+    const hostname = logHostname ? this.toStreamColor(this.hostname, Color.MAGENTA) + ' ' : '';
```

Comment:
that `EXPRESSION + ' ' ` is not very elegant, but we are focusing on performance here)

Reply:
I assume this is more performant that a template literal like

```ts
const hostname = logHostname ? `${this.toStreamColor(this.hostname)} ` : '';
```
?

Reply:
yeah, it's 4 times faster

Reply:
Dang! That's impressive. Sweet!